src/smart-components/group/add-group/summary-content.js

```diff
@@ -50,9 +50,9 @@ const SummaryContent = () => {
                 <GridItem md={9}>
                   <Text component={TextVariants.p}>
                     {selectedRoles.map((role, index) => (
-                      <Text className="pf-u-mb-0" key={index}>
+                      <div className="pf-u-mb-0" key={index}>
```

Comment:
Div cannot be inside P either. Maybe we should just remove the parent `p` component and use something like list?